config/optional/views.view.scheduled_transitions.yml

```diff
@@ -0,0 +1,578 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - dynamic_entity_reference
+    - scheduled_transitions
+    - user
+_core:
+  default_config_hash: CRJ4wWdCQQzLovoZ1qzwF5h0wcg1wOuGxdyX8Ma75Fk
```

Comment:
Does this default config hash need to be removed?

Reply:
yes. forgot to remove.~ lol